e2e_test/streaming/bug_fixes/panic_on_interval_4575_comment.slt

```diff
@@ -0,0 +1,14 @@
+statement ok
+create table t (v1 interval);
+
+statement ok
+insert into t values (interval '24 h'), (interval '1 d');
+
+statement ok
+create materialized view mv1 as select distinct v1 from t;
+
+statement ok
+drop materialized view mv1;
```

Comment:
Let's also select from mv1 to check the result 😂

Reply:
added